packages/css-modules-flow-types-loader/package.json

```diff
@@ -13,8 +13,7 @@
     "node": ">=4"
   },
   "scripts": {
-    "build": "babel *.js -d dist --ignore *.test.js",
-    "prepublish": "cd ../../ && npm run prepublish"
+    "build": "babel *.js -d dist --ignore *.test.js"
```

Comment:
Currently not sure how this works with Lerna, so can you add `prepublish` here again. But feel free to create a github issue on this. : )